packages/vite/src/node/config.ts

```diff
@@ -947,15 +955,14 @@ async function loadConfigFromBundledFile(
 export function loadEnv(
   mode: string,
   envDir: string,
-  prefix = 'VITE_'
+  prefixes: string[]
```

Comment:
It seems that vite export `loadEnv` function from codebase in https://github.com/vitejs/vite/issues/1930#issuecomment-783747858. This PR makes a breaking change to it.